src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractCommon.cs

```diff
@@ -22,11 +23,27 @@ public class MessageContractConstants
         public const string customerElementName = "Customer_Name";
         public const string customerElementNamespace = "http://www.contoso.com";
         public const string customerElementValue = "Michael Jordan";
+        public const string extraValuesName = "OutOfBandData";
+        public const string extraValuesNamespace = "http://www.contoso.com";
     }
 
     public class MessageContractHelpers
     {
-        public static XmlDictionaryReader SetupMessageContractTests(bool isWrapped)
+        public static XmlDictionaryReader SetupMessageContractTestsAndGetResponseBodyReader(string methodName)
+        {
+            var inspector = SetupMessageContractTests(methodName);
```

Comment:
Nit: prefer we don't use 'var' for situations like this but name the actual type for clarity.